front/internals/webpack/webpack.config.js

```diff
@@ -102,27 +116,16 @@ const config = {
       },
       {
         test: /\.(svg|jpg|png|gif)$/,
-        loader: 'url-loader',
-        options: {
-          limit: 8192,
-        }
+        type: 'asset/resource',
       },
       {
         test: /\.(eot|ttf|woff|woff2)$/,
-        loader: 'file-loader',
-        options: {
-          name: '[name].[ext]',
-        }
+        type: 'asset',
       },
       {
         test: /\.htaccess/,
```

Comment:
You removed its import in the app and it's still here, I'm not super clear on what it is and how it should be used but it seems a bit inconsistent, do we have an htaccess file and do we need it ?

Reply:
I removed it from the app because we are no longer using the file loader but indeed, I don't see such a file so it's probably safe to remove this altoghether.

Reply:
Oops, my bad, we do have an htaccess file. I will leave this as is.

Reply:
Another correction, there is such a file but apparently is not relevant anymore soo, once again, deleting this 😅